packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/index.ts

```diff
@@ -10,14 +12,11 @@ export {
   FailedLockToken,
   ReleaseResult,
   RejectResult,
-} from "./api/models.js";
-export {
   PublishCloudEventOptions,
   PublishCloudEventsOptions,
   ReceiveCloudEventsOptions,
   AcknowledgeCloudEventsOptions,
   ReleaseCloudEventsOptions,
   RejectCloudEventsOptions,
-} from "./api/operations.js";
-export { EventGridClient, EventGridClientOptions } from "./EventGridClient.js";
+} from "./api/index.js";
```

Comment:
I merge the previous exports from different files in the api folder into exports from `api/index.js` directly. Let me know if you are okay with this ?

Reply:
I assume we'll need to update again once #1915 is merged?